lib/abide_dev_utils/cem/generate/reference.rb

```diff
@@ -459,7 +459,7 @@ def config_example_builder
 
             @control.title.nil? ? out_str.unshift("    #{@control.id.dump}:") : out_str.unshift("    #{@control.title.dump}:")
             out_str.unshift('  control_configs:')
-            out_str.unshift("#{@module_name}::config:")
+            out_str.unshift("#{@module_name.split('-')[1]}::config:")
```

Comment:
```suggestion
            out_str.unshift("#{@module_name.split('-').last}::config:")
```
Bit more idiomatic to use `#last` instead of an explicit index